stable/kibana/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 name: kibana
-version: 0.13.1
+version: 0.13.2
```

Comment:
let's bump the minor version rather than patch version as this is a relatively significant new feature to the chart.